lib/ansible/modules/cloud/docker/docker_swarm.py

```diff
@@ -253,6 +256,7 @@ def __init__(self):
         self.force_new_cluster = None
         self.remote_addrs = None
         self.join_token = None
+        self.force = None
```

Comment:
Ah, this one should not have been there. I guess this is a rebasing artefact. I'll create a PR to get rid of it.